Sources/NIOConcurrencyHelpers/NIOAtomic.swift

```diff
@@ -311,3 +311,9 @@ public final class NIOAtomic<T: NIOAtomicPrimitive> {
         }
     }
 }
+
+#if compiler(>=5.5) && canImport(_Concurrency)
+extension NIOAtomic: Sendable {
+
+}
+#endif
```

Comment:
This one can just be `extension NIOAtomic: NIOSendable` (`NIOSendable` is just a typealias to `Sendable` when it's available and `Any` when it isn't)

Reply:
That will involve depending on `NIOCore` though, is that something we're ok doing?

Reply:
Very good point. No, probably not.